src/main/java/core/basesyntax/Ball.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Ball {
+    private final String color;
+    private final int number;
+
+    public Ball() {
+        ColorSupplier color = new ColorSupplier();
+        this.color = color.getRandomColor();
+        this.number = new Random().nextInt(99) + 1;
+    }

```

Comment:
move it to getRandomBall method and use Ball(int number, String color) constructor
```suggestion
    public Ball() {
        ColorSupplier color = new ColorSupplier();
        this.color = color.getRandomColor();
        this.number = new Random().nextInt(99) + 1;
    }

```